docs/assembly.rst

```diff
@@ -202,6 +202,12 @@ In the grammar, opcodes are represented as pre-defined identifiers.
 +-------------------------+------+-----------------------------------------------------------------+
 | byte(n, x)              |      | nth byte of x, where the most significant byte is the 0th byte  |
 +-------------------------+------+-----------------------------------------------------------------+
+| shl(x, y)               |      | logical shift left y by x                                       |
```

Comment:
We should add comments that tell the user which EVM version is required.

Reply:
Do you mean just a commit in the description or a new column at some point?

Reply:
Either a new column or just in parentheses.

Reply:
Thinking about reusing the middle column. It currently is used for *empty* (pushes one item), `-` (doesn't push anything onto the stack), `*` (special). It could just have `B` for byzantium ones?

Reply:
Will add the `C` after #3604 is merged.